sdk/servicebus/service-bus/test/public/atomManagement.spec.ts

```diff
@@ -2185,777 +1821,1269 @@ describe(`createRule() using different variations to the input parameter "ruleOp
+      }
+      case EntityType.SUBSCRIPTION: {
+        if (!topicPath) {
+          throw new Error(
+            "TestError: Topic path must be passed when invoking tests on subscriptions"
+          );
+        }
+        const subscriptionResponse = await atomClient.deleteSubscription(topicPath, entityPath);
+        return subscriptionResponse;
+      }
+      case EntityType.RULE: {
+        if (!topicPath || !subscriptionPath) {
+          throw new Error(
+            "TestError: Topic path AND subscription path must be passed when invoking tests on rules"
+          );
+        }
+        const ruleResponse = await atomClient.deleteRule(topicPath, subscriptionPath, entityPath);
+        return ruleResponse;
+      }
     }
-    case EntityType.SUBSCRIPTION: {
-      if (!topicPath) {
-        throw new Error(
-          "TestError: Topic path must be passed when invoking tests on subscriptions"
+    throw new Error("TestError: Unrecognized EntityType");
+  }
+
+  async function listEntities(
+    testEntityType: EntityType,
+    topicPath?: string,
+    subscriptionPath?: string,
+    skip?: number,
+    maxCount?: number
+  ): Promise<any> {
+    switch (testEntityType) {
+      case EntityType.QUEUE: {
+        const queueResponse = await serviceBusAtomManagementClient["getQueues"]({
+          skip,
+          maxCount
+        });
+        return queueResponse;
+      }
+      case EntityType.TOPIC: {
+        const topicResponse = await serviceBusAtomManagementClient["getTopics"]({
+          skip,
+          maxCount
+        });
+        return topicResponse;
+      }
+      case EntityType.SUBSCRIPTION: {
+        if (!topicPath) {
+          throw new Error(
+            "TestError: Topic path must be passed when invoking tests on subscriptions"
+          );
+        }
+        const subscriptionResponse = await serviceBusAtomManagementClient[
+          "getSubscriptions"
+        ](topicPath, { skip, maxCount });
+        return subscriptionResponse;
+      }
+      case EntityType.RULE: {
+        if (!topicPath || !subscriptionPath) {
+          throw new Error(
+            "TestError: Topic path AND subscription path must be passed when invoking tests on rules"
+          );
+        }
+        const ruleResponse = await serviceBusAtomManagementClient["getRules"](
+          topicPath,
+          subscriptionPath,
+          { skip, maxCount }
         );
+        return ruleResponse;
       }
-      const subscriptionResponse = await serviceBusAtomManagementClient.deleteSubscription(
-        topicPath,
-        entityPath
-      );
-      return subscriptionResponse;
     }
-    case EntityType.RULE: {
-      if (!topicPath || !subscriptionPath) {
-        throw new Error(
-          "TestError: Topic path AND subscription path must be passed when invoking tests on rules"
-        );
+    throw new Error("TestError: Unrecognized EntityType");
+  }
+
+  describe.only("Premium Namespaces", () => {
```

Comment:
`.only` left behind.

Reply:
This was purposeful, wanted to check if my changes to arm template succeed in the CI.

Will remove after the test run.